modules/prebidServerBidAdapter/index.js

```diff
@@ -814,6 +814,19 @@ Object.assign(ORTB2.prototype, {
         if (adapter && adapter.getSpec().transformBidParams) {
           bid.params = adapter.getSpec().transformBidParams(bid.params, true, adUnit, bidRequests);
         }
+        // passing bid.bidViewability to pubmatic params, only when present
+        const addBidViewabilityDataS2S= function (bidRequests,bid){
+          for (let i = 0; i < bidRequests.length; i++) {  
```

Comment:
try to use forEach for better testability

Reply:
used forEach

Reply:
used forEach as per comment